loadtest/report/output.go

```diff
@@ -167,3 +168,7 @@ func sortKeys(m map[model.LabelValue]avgp99) []model.LabelValue {
 	})
 	return labels
 }
+
+func roundTo3DecimalPlaces(f float64) float64 {
```

Comment:
That's some explicit naming :grinning:

Reply:
It's the hardest problem in computer science for a reason. :)